src/test/java/io/stargate/sgv2/jsonapi/api/v1/VectorSearchIntegrationTest.java

```diff
@@ -803,7 +802,7 @@ public void happyPathWithEmptyVector() {
           .body("errors", hasSize(1))
           .body("errors[0].exceptionClass", is("JsonApiException"))
           .body("errors[0].errorCode", is("SHRED_BAD_VECTOR_SIZE"))
-          .body("errors[0].message", is(ErrorCode.SHRED_BAD_VECTOR_SIZE.getMessage()));
+          .body("errors[0].message", is("$vector value can't be empty"));
```

Comment:
Not sure why this was changed? Earlier code more robust wrt changes to error message.

Reply:
Because I added `extendError` - the configuration in the `ErrorCode` class. It will cause the class initialization error in this case - because it cannot find the config

Reply:
Ok. We will probably change things to ensure that no loading occurs from within static initialization of `ErrorCode` (or its replacement) in future.
But is fine for now.